content/blog/mlops-–-deploying-an-ml-model-in-greenlake-platform-mlops-service.md

```diff
@@ -0,0 +1,953 @@
+	{mso-style-name:"Table Normal";
+	mso-tstyle-rowband-size:0;
+	mso-tstyle-colband-size:0;
+	mso-style-noshow:yes;
+	mso-style-priority:99;
+	mso-style-parent:"";
+	mso-padding-alt:0in 5.4pt 0in 5.4pt;
+	mso-para-margin-top:0in;
+	mso-para-margin-right:0in;
+	mso-para-margin-bottom:8.0pt;
+	mso-para-margin-left:0in;
+	line-height:107%;
+	mso-pagination:widow-orphan;
+	font-size:11.0pt;
+	font-family:"Calibri",sans-serif;
+	mso-ascii-font-family:Calibri;
+	mso-ascii-theme-font:minor-latin;
+	mso-hansi-font-family:Calibri;
+	mso-hansi-theme-font:minor-latin;
+	mso-bidi-font-family:"Times New Roman";
+	mso-bidi-theme-font:minor-bidi;}
+</style>
+<!\\[endif]-->
+
+<!--StartFragment-->
+
+**Overview**
+
+HPE GreenLake Central is an advanced software-as-a-service platform that provides you with a consistent cloud experience for all your applications and data—on-premises or off-premises. It provides you with insights and controls to manage your hybrid IT estate, complementing your use of public clouds and data centers. HPE GreenLake Central gives you the ability to choose where and how to place your workloads and data, and—through the services you purchase—enables you to monitor security, capacity, resource utilization, and costs.
+
+HPE GreenLake for ML Ops is an on-premises, enterprise-grade ML service, enabling developers and data scientists to rapidly build, train, and deploy ML models—from pilot to production, at any scale.
+
+This preconfigured solution comprises an optimized hardware stack and is powered by HPE Ezmeral Container Platform. It provides data scientists with self-service access to a sandbox environment for prototyping and testing, to eliminate IT provisioning delays, ensure repeatability, and accelerate time-to-value. As a fully managed solution, the HPE GreenLake for ML Ops offering frees IT from routine infrastructure management tasks.
+
+**Machine learning Lifecycle**
+
+![ML Life cycle](/img/blog_3.png "ML Life cycle")
+
+The ML project life cycle can generally be divided into three main stages: data preparation, model creation, and deployment. All three of these components are essential for creating quality models that will bring added value to your business. This process is *cyclical* because the insights gained from the existing model will help define the next model to be deployed.
+
+In this article, we focus on optimal model identified after data preparation and building model. More specifically, as an example, we shall deploy a sample ONNX model available in Kubernetes native MinIO Object storage, using Triton Inference Server.
+
+Triton Inference server supports deployment of any AI model from multiple deep learning and machine learning frameworks, including TensorRT, TensorFlow, PyTorch, ONNX, OpenVINO, Python, RAPIDS FIL, and more. Triton delivers optimized performance for many query types, including real time, batched, ensembles and audio/video streaming. For more refer the references section below
```

Comment:
To learn more about Triton Inference server, refer to the References section at the end of the post.